ci/docker/base.dockerfile

```
@@ -0,0 +1,130 @@
+# Base dockerfile to build images used in Darma testing.
+
+ARG ARCH=amd64
+ARG BASE=ubuntu:22.04
+ARG SETUP_ID=${SETUP_ID:-"ubuntu-develop"}
+ARG CI=${CI:-""}
+
+ARG CC=${CC:-""}
+ARG CXX=${CXX:-""}
+ARG FC=${FC:-""}
+ARG GCOV=${GCOV:-""}
+
+ARG PROXY=${http_proxy:-""}
+
+ARG CPATH=${CPATH:-""}
+ARG INFOPATH=${INFOPATH:-""}
+ARG LD_LIBRARY_PATH=${LD_LIBRARY_PATH:-""}
+ARG LIBRARY_PATH=${LIBRARY_PATH:-""}
+
+ARG CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-""}
+ARG CMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD:-""}
+ARG CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH:-""}
+
+ARG CCACHE_COMPILERCHECK
+ARG CCACHE_COMPRESS
+ARG CCACHE_COMPRESSLEVEL
+ARG CCACHE_MAXSIZE
+ARG CCACHE_DIR
+
+ARG MPICH_CC=${MPICH_CC:-""}
+ARG MPICH_CXX=${MPICH_CXX:-""}
+
+ARG ZOLTAN_INSTALL_DIR=/opt/trilinos/bin
+
+# Add to path optional dependencies path
+ARG PATH_PREFIX=${PATH_PREFIX:-""}
+
+## Specific (Intel)
+ARG CMPLR_ROOT \
+    INTEL_LICENSE_FILE \
+    ONEAPI_ROOT \
+    TBBROOT
+
+FROM --platform=${ARCH} ${BASE} as base
```

```suggestion
FROM --platform=${ARCH} ${BASE} AS base
```

IIRC docker complains about lowercase `as`.

---

Thanks this has been updated.